openy_home_branch/modules/ws_home_branch/assets/js/ws-header-lb-selector.js

```diff
@@ -34,13 +35,18 @@
         }
         let selected = Drupal.homeBranch.getValue('id');
         let locations = Drupal.homeBranch.getLocations();
+        console.log(Drupal.homeBranch);
         if (selected) {
           this.element.text(locations[selected]);
           this.element.parent().addClass('selected');
+          this.element.attr('href', '/node/' + Drupal.homeBranch.getValue('id'));
```

Comment:
we need to ensure code works in subfolder too ( when the site is example.com/subsite/index.php